src/libstd/keyword_docs.rs

```diff
@@ -352,6 +352,7 @@ mod fn_keyword { }
 /// For more information on for-loops, see the [Rust book] or the [Reference].
 ///
 /// [`impl`]: keyword.impl.html
+/// [higher-ranked trait bounds]: https://doc.rust-lang.org/nightly/reference/trait-bounds.html#higher-ranked-trait-bounds
```

Comment:
Should this not have `nightly/`?

Reply:
Yeah you can remove that.

Reply:
It's too long even if I remove it.  Do I just put the link on the next line?  I'm not sure how Markdown would handle that.

Reply:
I think moving it down should work

Reply:
Oh yeah literally 2 lines down it does that lol